taxi/forms.py

```diff
@@ -1,16 +1,45 @@
 from django import forms
 from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm
 from django.core.exceptions import ValidationError
 
 from taxi.models import Driver, Car
 
 
+class DriverLicenseCreateForm(UserCreationForm):

```

Comment:
it is not a License, but DriverCreation form. When Driver creates - what makes him a "driver" actually?
Also, now only this form has validation, what about UpdateLicenseForm?